db/db_test.cc

```diff
@@ -7,10 +7,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
+// Introduction of SyncPoint effectively disabled building and running this test in Release build.
+// which is a pity, it is a good test
+#if !(defined NDEBUG) || !defined (OS_WIN)
```

Comment:
@igorcanadi Understood. However, just FYI, on Windows Release build perf diff is about 10x with Debug and that affects perf related tests. Also on one occasion we had test pass in Debug but not in Release.